components/automate-chef-io/content/docs/migrate.md

```diff
@@ -113,6 +113,7 @@ If you rely on any of the capabilities listed below, we recommend you continue t
 Chef continues support of Chef Automate 1 while we develop upgrade paths for these capabilities on the new platform.
```

Comment:
No longer true--we do not support Automate 1 any longer (as of 12/31/2019).

Reply:
@msorens can you open a new pr for this?